hyperion-sqlite/src/main/java/com/willowtreeapps/hyperion/sqlite/presentation/database/DatabaseListActivity.java

```diff
@@ -0,0 +1,55 @@
+package com.willowtreeapps.hyperion.sqlite.presentation.database;
+
+import android.os.Bundle;
+import android.support.annotation.Nullable;
+import android.support.v7.app.ActionBar;
+import android.support.v7.app.AppCompatActivity;
+import android.support.v7.widget.LinearLayoutManager;
+import android.support.v7.widget.RecyclerView;
+import android.support.v7.widget.Toolbar;
+
+import com.willowtreeapps.hyperion.sqlite.R;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+
+public class DatabaseListActivity extends AppCompatActivity {
```

Comment:
should add `@HyperionIgnore` annotation to this Activity, so the menu is not embedded into Plugin activities